src/js/ui/element.js

```diff
@@ -0,0 +1,47 @@
+/**
+ *
+ * @param {string} selector
+ * @returns {HTMLElement}
+ */
+export const querySelector = (selector) => document.querySelector(selector);
+
+/**
+ *
+ * @param {string} selector
+ * @returns {HTMLElement[]}
+ */
+export const querySelectorAll = (selector) => document.querySelector(selector);
```

Comment:
```suggestion
/**
 *
 * @param {string} selector
 * @returns {HTMLElement}
 */
export const querySelector = (selector, parent = document) => parent.querySelector(selector);

/**
 *
 * @param {string} selector
 * @returns {HTMLElement[]}
 */
export const querySelectorAll = (selector, parent = document) => parent.querySelector(selector);
```
이렇게 parent도 받아올 수 있어야 더 유연하게 사용할 수 있을 것 같아요~